sensing/pointcloud_preprocessor/config/concatenate_pointclouds_param_file.yaml

```diff
@@ -0,0 +1,8 @@
+/**:
+  ros__parameters:
+    ## Paramters in concatenate_pointclouds.cpp
+    output_frame: ""
+    input_topics: [""]
+    max_queue_size: 5
+    timeout_sec: 0.033
+    input_offset: {}
```

Comment:
Why this is {} instead of []

Reply:
Thank you! I will revise all of them. For this one, [] does not work and will report error. The original code uses {}, so I just adapted {}.